StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -93,6 +93,120 @@ extension EmbeddedPaymentElement: EmbeddedPaymentMethodsViewDelegate {
     func selectionDidUpdate() {
         delegate?.embeddedPaymentElementDidUpdatePaymentOption(embeddedPaymentElement: self)
     }
+    func presentSavedPaymentMethods(selectedSavedPaymentMethod: STPPaymentMethod?) {
+        let elementsSession = loadResult.elementsSession
+        if savedPaymentMethods.count == 1,
+           let paymentMethod = savedPaymentMethods.first,
+           paymentMethod.isCoBrandedCard,
+           elementsSession.isCardBrandChoiceEligible {
+            let updateViewController = UpdateCardViewController(paymentMethod: paymentMethod,
+                                                                removeSavedPaymentMethodMessage: configuration.removeSavedPaymentMethodMessage,
+                                                                appearance: configuration.appearance,
+                                                                hostedSurface: .paymentSheet,
+                                                                canRemoveCard: configuration.allowsRemovalOfLastSavedPaymentMethod && elementsSession.allowsRemovalOfPaymentMethodsForPaymentSheet(),
+                                                                isTestMode: configuration.apiClient.isTestmode,
+                                                                cardBrandFilter: configuration.cardBrandFilter)
+            updateViewController.delegate = self
+            let bottomSheetVC = BottomSheetViewController(contentViewController: updateViewController,
+                                                          appearance: configuration.appearance,
+                                                          isTestMode: configuration.apiClient.isTestmode, didCancelNative3DS2: {
+                self.paymentHandler.cancel3DS2ChallengeFlow()
```

Comment:
I think it makes more sense to no-op or assertionFailure or something here, why would update invoke 3ds2?

Reply:
(It's very stinky that BottomSheetVC has this parameter, I have a ticket somewhere to refactor this)